packages/mm-backend/src/modules/mixin/snapshots/snapshots.service.ts

```diff
@@ -0,0 +1,68 @@
+import { Injectable } from '@nestjs/common';
+import { MixinGateway } from '../../../integrations/mixin.gateway';
+import { Cron } from '@nestjs/schedule';
+import { CustomLogger } from '../../logger/logger.service';
+import { StrategyExecutor } from '../../../common/utils/trading-dispatcher/strategy-executor';
+import { SafeSnapshot } from '@mixin.dev/mixin-node-sdk';
+import { SnapshotsRepository } from './snapshots.repository';
+
+@Injectable()
+export class SnapshotsService {
```

Comment:
What's that?

Reply:
It's excluded to another branch: Fork-issue_17-snapshot-processing